deployment/ccip/changeset/deploy.go

```diff
@@ -67,25 +61,62 @@ var (
 	USDCTokenPool       deployment.ContractType = "USDCTokenPool"
 )
 
-func DeployPrerequisiteChainContracts(e deployment.Environment, ab deployment.AddressBook, selectors []uint64) error {
+type DeployPrerequisiteContractsOpts struct {
+	USDCEnabledChains []uint64
+	Multicall3Enabled bool
+}
+
+type PrerequisiteOpt func(o *DeployPrerequisiteContractsOpts)
+
+func WithUSDCChains(chains []uint64) PrerequisiteOpt {
```

Comment:
these still don't look very useful to me, whats wrong with just using the struct directly?